### ISSUE TITLE
Failed to compile

### ISSUE THREAD

**Issue by Petrosvj** [eu-digital-green-certificates/dgca-issuance-web#147]
./src/assets/SCSS/index.scss (./node_modules/css-loader/dist/cjs.js??ref--5-oneOf-6-1!./node_modules/postcss-loader/src??postcss!./node_modules/resolve-url-loader??ref--5-oneOf-6-3!./node_modules/sass-loader/dist/cjs.js??ref--5-oneOf-6-4!./src/assets/SCSS/index.scss)
Node Sass does not yet support your current environment: Windows 64-bit with Unsupported runtime (93)
For more information on which environments are supported please see:
https://github.com/sass/node-sass/releases/tag/v5.0.0

i ran yarn from git bash and gave me this error when opening the web page at the end of computing

**Comment by TEXLUCA87** [comment]
I have the same problem

**Comment by Petrosvj** [comment]
> I have the same problem win 32 or 64 bit. Both.

yeah, i tried both too and it didn't change anything... i even tried older versions of Node, downloading updates and Chocolately but nothing changed. I think they blocked the project for the attention they received because developers aren't even answering to issues nor working on the project i guess...

**Comment by legrandse** [comment]
You need to install node.js 15.0.0

**Comment by Petrosvj** [comment]
> You need to install node.js 15.0.0

even with node.js version 15.0.0 the error "failed to compile" comes out.
in order to fix it instead of writing "$ npm install -g yarn" you must write "$ npm rebuild node-sass".
the program will say "compiled succesfully" and open correctly the internet page but will keep loading it never being able to show the page... i have a couple of error messages before the "complied succesfully" ending, but i don't think they matter anything

**Comment by steeven93** [comment]
> 

io ci sono riuscito e fatto, vi spiego il passaggio

vi scaricate la versione 1.0.10
poi aprite il pompt dei comandi

entrate nella cartella del progetto 

`cd <nome-progetto>`
una volta dentro , assicuratevi di avere docker installato, altrimenti scaricatelo
una volta installato docker il passaggio successivo sarà quello di creare VM con docker
quindi digiterete questo:

`docker-compose up -d`

aspettate un pò perché vi dovrà scaricare tutta la roba sul container virtuale che vi creerà.

una volta fatto questo aprite la dashaboard di docker e andate su **Containers/APP** 
lì troverete un contenitore chiamato **dgca-issuance-web-1010**
se siete arrivati fin qui ... bene! siamo a metà strada.
se cliccate sul contenitore vi si aprira una lista di sub contenitori .
e ci concentreremo su **dgca-issuance-web** e vedrete delle icone, premete sull'icona **>_** , vi si aprirà il terminale.
se provate ad accedere su localhost:8080 vi renderete conto che non vi fa entrare perché vi chiederà una password.

che faremo noi? beh, creeremo un nuovo utente quindi nel terminale che vi si è aperto in precedenza cliccando su **>_** dobbiamo installare questo _**apache2-utils**_ e lo installeremo con i comandi di linux, in questo modo :
`apt-get install apache2-utils`
molto bene! 
ora basterà creare un utente nostro .
sul terminale scrivete questo : 
`htpasswd -c /etc/nginx/.htpasswd nome-utente` 
ovviamente il nome-utente va sostituito con quello che voi volete , ok ?
una volta fatto partire il comando vi chiederà di inserire una password, e vi richiederà di inserirla nuovamente.

fatto ciò , ragazzi è fatta! potete accedere a localhost:8080 e inserirete i vostri dati utente come le avete creati , accedete e il piatto è pronto, purtroppo il codice qr non è valido , qualcuno ha delle idee?

**Comment by Petrosvj** [comment]
> 

nel tutorial degli sviluppatori non è richiesto Docker che a me da solo problemi e non funziona (Docker engine failed to start)... probabilmente il QR code generato non funziona per via dell'update in vista del super green pass o perchè c'è un database di conferma dei dati, non ne ho idea...

**Comment by steeven93** [comment]
> > 
> 
> 
> 
> nel tutorial degli sviluppatori non è richiesto Docker che a me da solo problemi e non funziona (Docker engine failed to start)... probabilmente il QR code generato non funziona per via dell'update in vista del super green pass o perchè c'è un database di conferma dei dati, non ne ho idea...

Vero non è richiesto perché è solo il frontend, è il front end non genera i qr, quindi ci vuole il back end perché non día errore 500 nella creazione ... all'interno del file c'è un file docker per la creazione di tutto l'ambiente .  Prova a installare la versione 3.0 di Docker

**Comment by Petrosvj** [comment]
> 

come detto da un altro utente sicuramente serve la conferma del back end o l'autorizzazione per avere un QR valido, specialmente ora... 
per me nulla da fare, a quanto pare il mio processore non è supportato per Docker... qualsiasi versione io scarichi mi da un errore, ho un msi GTX970, i7 6700 con Windows 10. 
Ho provato con più stringhe ad aprire il file con Docker ma gli errori che mi da su Git Bash sono:

> docker: error during connect: In the default daemon configuration on Windows, the docker client must be run with elevated privileges to connect.: Post http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.24/containers/create: open //./pipe/docker_engine: Impossibile trovare il file specificato.
See 'docker run --help'.

> [6380] Failed to execute script docker-compose
Traceback (most recent call last):
  File "site-packages\docker\api\client.py", line 205, in _retrieve_server_version

>docker.errors.DockerException: Error while fetching server API version: (2, 'CreateFile', 'Impossibile trovare il file specificato.')

**Comment by Petrosvj** [comment]
> Vero non è richiesto perché è solo il frontend, è il front end non genera i qr, quindi ci vuole il back end perché non día errore 500 nella creazione ... all'interno del file c'è un file docker per la creazione di tutto l'ambiente . Prova a installare la versione 3.0 di Docker

Docker non mi funzionava perchè non avevo aggiornato WSL, se anche dopo aver aggiornato WSL non vi parte l'engine andate in %Appdata%\Docker ed eliminate il file Settings.JSON aprite poi Docker eseguendolo come amministratore.
Nonostante ciò quando eseguo il comando > docker-compose up -d
mi esce questo errore:

Error response from daemon: Head "https://docker.pkg.github.com/v2/eu-digital-green-certificates/dgca-issuance-service/dgca-issuance-service/manifests/1.0.3": no basic auth credentials

l'errore mi esce anche dopo aver fatto il pull della versione 1.0.10... hai provato il procedimento con l'ultima versione?
